### PR TITLE
Calculate Text object box size correctly

### PR DESCRIPTION
**Pull request by tobbi**

### CODE REVIEW

**Comment by maxteufel** [comment]
As I said on IRC, splitting the text in multiple lines so it fits the current box would be preferred.

**Comment by maxteufel** [comment]
If this is supposed to be for 0.5.1, please close this PR and open one on the release/0.5 branch.

File: src/object/text_object.cpp [hunk 1]
```diff
@@ -103,8 +101,9 @@ TextObject::draw(DrawingContext& context)
     return;
   }
 
-  float width  = 500;
-  float height = 70;
+  // 20 = twice the padding in the draw_text call below
+  float width  = font->get_text_width(text) + 20;
+  float height = font->get_text_height(text) + 20;
```
Comment (Hume2) [inline src/object/text_object.cpp]: Maybe it would be better if the size was precalculated on set_text.

**Comment by tobbi** [comment]
Superseded.

**Pull request closed without merging**